src/reports_api/models/code_table.py

```diff
@@ -43,6 +45,29 @@ def find_all(cls):
         codes = cls.query.all()  # pylint: disable=no-member
         return codes
 
+    @staticmethod
```

Comment:
These methods doesn't have to be added to CodeTable. It was intentional, as code tables won't be updated/created/deleted programatically.
For any model which is not code table, can set base class as BaseModel which would inherit CRUD methods to that model.

Reply:
Removed the methods and added missing methods to BaseModel